tap_looker/transform.py

```diff
@@ -8,6 +8,9 @@
 # Loop through and replace $ref references in nested dict and lists
 def replace_refs(this_dict, swagger):
     for k, v in list(this_dict.items()):
+        if k == 'properties' and isinstance(this_dict, dict):
```

Comment:
`and isinstance(this_dict, dict)`

Correct me if I'm wrong, but this check is pointless since we already call `this_dict.items()` in the previous line.

Reply:
I think this could still be a list/array or a dict. I am going to keep it to be safe.